Wavefront.SDK.CSharp.Test/ApplicationTagsTest.cs

```diff
@@ -46,5 +47,52 @@ public void TestToPointTags()
             Assert.Contains(new KeyValuePair<string, string>("env", "production"), pointTags);
             Assert.Contains(new KeyValuePair<string, string>("location", "SF"), pointTags);
         }
+
+        [Fact]
+        public void TestTagsFromEnv()
+        {
+            SetEnvVar();
+            var pointTags = new ApplicationTags.Builder("app1", "service1")
+                .CustomTags(new Dictionary<string, string>
+                {
+                    { "env", "production" },
+                    { "location", "SF" }
+                })
+                .tagsFromEnv("MY*")
```

Comment:
`MY.*` or `^MY.*$` would be better.  `MY*` only matches `M`, `MY`, `MYY`, etc.  The test is still passing because the C# Regex.Match matches anywhere in the input string (whereas Java Matcher.match must match the entire string).  BTW, I think the difference in behavior is ok because Java and C# devs should be familiar with how the matching works in their respective languages and can adjust their patterns accordingly.